upstairs/src/volume.rs

```diff
@@ -4501,101 +5030,21 @@ mod test {
         n_opts.target[1] = "127.0.0.1:8888".parse().unwrap();
         n_opts.read_only = true;
 
-        assert!(test_volume_replace_opts(
-            vol_id,
-            512,
-            blocks_per_extent,
-            extent_count,
-            o_opts,
-            n_opts
-        )
-        .is_err());
-    }
-
-    #[test]
-    fn volume_replace_with_rop() {
```

Comment:
Was this test folded into somewhere else?

Reply:
I think it may have been a duplicate before this PR, but for sure we cover it in the `volume_replace_basic()` tests